app/authenticated/project/api/hooks/route.js

```diff
@@ -13,6 +13,8 @@ export default Ember.Route.extend({
 
       return Ember.RSVP.hash({
         receivers: this.get('webhookStore').findAll('receiver', {forceReload: true}),
+      }).then(({receivers}) => {
+        return receivers.filter((ele) => ele.driver !== 'forwardPost');
```

Comment:
This makes it a static array instead of a live one that will update when receivers come and go while the page is up.  Filter as a computed property in the controller instead of the route.

Reply:
@vincent99 I've updated it.